libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -215,55 +232,61 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
   @Output()
   // eslint-disable-next-line @angular-eslint/no-output-native
   public get change(): Observable<SkyRadioChange> {
-    return this._change;
+    return this.#_change;
   }
 
   /**
    * Fires when the selected value changes.
    */
   @Output()
   public get checkedChange(): Observable<boolean> {
-    return this._checkedChange;
+    return this.#_checkedChange;
   }
 
   /**
    * Fires when the selected value changes.
    */
   @Output()
   public get disabledChange(): Observable<boolean> {
-    return this._disabledChange;
-  }
-
-  public get inputId(): string {
-    return `sky-radio-${this.id}-input`;
+    return this.#_disabledChange;
   }
 
   public set selectedValue(value: any) {
-    if (value !== this._selectedValue) {
-      this._selectedValue = value;
+    if (value !== this.#_selectedValue) {
+      this.#_selectedValue = value;
     }
   }
   public get selectedValue(): any {
-    return this._selectedValue;
+    return this.#_selectedValue;
   }
 
+  public inputId = '';
+
   public radioGroupDisabled = false;
 
-  private _change = new EventEmitter<SkyRadioChange>();
-  private _checked = false;
-  private _checkedChange = new BehaviorSubject<boolean>(this._checked);
-  private _disabled = false;
-  private _disabledChange = new BehaviorSubject<boolean>(this._disabled);
-  private _name: string;
-  private _radioType: string;
-  private _selectedValue: any;
-  private _tabindex: number;
-  private _value: any;
+  #defaultId = `sky-radio-${++nextUniqueId}`;
+
+  #_change = new EventEmitter<SkyRadioChange>();
+  #_checked = false;
+  #_checkedChange = new BehaviorSubject<boolean>(this.#_checked);
+  #_disabled = false;
+  #_disabledChange = new BehaviorSubject<boolean>(this.#_disabled);
+  #_name: string | undefined;
+  #_radioType = 'info';
+  #_selectedValue: any;
+  #_tabindex = 0;
+  #_value: any;
```

Comment:
Is there a better type for this?

Reply:
Could we get away with using `unknown` here?

Reply:
Updated to `unknown`